docs/src/components/Layout/Sidebar.tsx

```diff
@@ -183,8 +186,8 @@ const SecondaryNav = (props) => {
         ))}
       </ExpanderItem>
 
-      {/* Flutter doesn't have legacy components */}
-      {platform === 'flutter' ? null : (
+      {/* Flutter and React Native don't have legacy components */}
+      {isFlutter || isReactNative ? null : (
```

Comment:
nit: Could just do `{!(isFlutter || isReactNative) && <ExpanderItem...` instead of the ternary operator. Same nit for below usages.

Reply:
Tbh i prefer the ternary returning `null` over the usage of `false` in the rendering tree

Reply:
+1 to returning null, the && can sometimes end up printing `0` in the dom